core/app/components/FormBuilder/actions.ts

```diff
@@ -9,10 +9,18 @@ import { logger } from "logger";
 import type { FormBuilderSchema } from "./types";
 import { db } from "~/kysely/database";
 import { isUniqueConstraintError } from "~/kysely/errors";
+import { getLoginData } from "~/lib/authentication/loginData";
+import { ApiError } from "~/lib/server";
 import { autoRevalidate } from "~/lib/server/cache/autoRevalidate";
 import { defineServerAction } from "~/lib/server/defineServerAction";
 
 export const saveForm = defineServerAction(async function saveForm(form: FormBuilderSchema) {
+	const loginData = await getLoginData();
+
```

Comment:
I think these form builder actions should actually just use the `editCommunity` capability. I don't think the form permissions are actually relevant for these actions - they're more for authorizing users to edit/create pubs with specific forms when they might otherwise not have any pub edit/create permissions in the community.

Reply:
sounds good, I've added `editCommunity` to each of these